draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -254,11 +254,12 @@ this context by a 4-tuple of source and destination address/ port pairs.
 
 Subflow: A flow of DCCP segments operating over an individual path,
 which forms part of a larger MP-DCCP connection. A subflow is started
-and terminated similar to a regular (single-path) DCCP connection.
+and terminated similar to a regular (single-path) DCCP connection. The term subflow
+can also be used to refer to an MP-DCCP connection with a single path.
 
 (MP-DCCP) Connection: A set of one or more subflows, over which an
 application can communicate between two hosts. There is a one-to-one
-mapping between a connection and an application socket.
+mapping between a connection and an MP-DCCP connection socket.
```

Comment:
This does not read so well. Also not sure the MP-DCCP features need to be/are exposed to the application? Possibly this should be something like "The MP-DCCP connection is exposed as a single DCCP socket to the application." ?

But possibly we can leave it for now and open a new issue.

Reply:
Incorporated https://github.com/markusa/ietf-multipath-dccp/pull/97/commits/d86db59cedaeefde38311dd6846ae93625dd9cb8